testing/internal-integ-testing/src/main/groovy/org/gradle/test/preconditions/IntegTestPreconditions.groovy

```diff
@@ -299,13 +299,6 @@ class IntegTestPreconditions {
         }
     }
 
-    static class JavaHomeWithDifferentVersionAvailable implements TestPrecondition {
```

Comment:
moving to cohesion as now tightly coupled, kind of making **fusion candidate** smell, right?